### PR TITLE
[IMP] base: cover filestore gc with tests

### PR DESCRIPTION
**Pull request by Julien00859**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by Julien00859** [comment]
@rco-odoo c'était beaucoup plus facile à tester que ça en avait l'air :p

File: odoo/addons/base/tests/test_ir_attachment.py [hunk 1]
```diff
@@ -225,6 +227,34 @@ def test_11_copy(self):
...
+    def test_13_rollback(self):
+        self.registry.enter_test_mode(self.cr)
+        self.addCleanup(self.registry.leave_test_mode)
+        self.cr = self.registry.cursor()
+        self.addCleanup(self.cr.close)
+        self.env = odoo.api.Environment(self.cr, odoo.SUPERUSER_ID, {})
+
+        # the data needs to be unique so that no other attachment link
+        # the file so that the gc removes it
+        unique_blob = uuid.uuid4().bytes
```
Comment (xmo-odoo) [inline odoo/addons/base/tests/test_ir_attachment.py:250]: Just go get bytes from `os.urandom`, there's no reason to invoke the additional processing layers of uuid if the end-goal is to get random data (even less so as UUIDs discard 6 bits to encode the version and variant, so this is strictly less unique than just calling `os.urandom(16)`).

**Review by rco-odoo — APPROVED** [review]
Very good!

**Comment by rco-odoo** [comment]
@robodoo override=ci/security

It's only a unit test calling `os.random()` to get some "unique" blob content.

**Comment by rco-odoo** [comment]
@robodoo r+

**Pull request closed without merging**